### PR TITLE
Release/2.0.1

### PR DESCRIPTION
**Pull request by mdeceglie**
- ~Code changes are covered by tests~
-  ~New functions added to `__init__.py`~
- ~API.rst is up to date, along with other sphinx docs pages~
- [x] Example notebooks are rerun and differences in results scrutinized
- [x] Updated changelog

### CODE REVIEW

**Comment by mdeceglie** [comment]
Ran the notebooks, no changes relative to development branch

File: docs/sphinx/source/changelog/v2.0.1.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+*************************
+v2.0.1 (October 30, 2020)
+*************************
+
+Deprecations
+------------
+* The the deprecation of :py:func:`~rdtools.normalization.pvwatts_dc_power`,
```
Comment (kanderso-nrel) [inline docs/sphinx/source/changelog/v2.0.1.rst:7]: Oops, missed double "the" in the review.  Also has an unnecessary comma at the end of this line

**Review by kanderso-nrel — COMMENTED** [review]
LGTM pending CI pass. Might not get test results back until Travis can catch up on its backlog over the weekend: https://www.traviscistatus.com/#system-metrics

Also looks like we forgot to squash when merging the last couple PRs?  NBD, just pointing it out for future merges.

**Review by kanderso-nrel — APPROVED** [review]
LGTM

**Pull request merged**